src/components/Admin/Admin.js

```diff
@@ -0,0 +1,134 @@
+import {React, Component} from 'react';
+import {Table, Tag, Space} from 'antd';
+
+/*Placeholder data*/
+const columns = [
+    {
+      title: 'First Name',
+      dataIndex: 'fname',
+      filters: [
+        {
+          text: 'Joe',
+          value: 'Joe'
+        },
+        {
+          text: 'Jim',
+          value: 'Jim'
+        }
+      ],
+      onFilter: (value, record) => record.fname.indexOf(value) === 0,
+      key: 'fname',
+      render: text => <a>{text}</a>,
+    },
+    {
+      title: 'Last Name',
+      dataIndex: 'lname',
+      filters: [
+        {
+          text: 'Brown',
+          value: 'Brown'
+        },
+        {
+          text: 'Green',
+          value: 'Green'
+        },
+        {
+          text: 'Black',
+          value: 'Black'
+        }
+      ],
+      onFilter: (value, record) => record.lname.indexOf(value) === 0,
+      key: 'lname',
+      render: text => <a>{text}</a>
+    },
+    {
+      title: 'Age',
+      dataIndex: 'age',
+      key: 'age',
+    },
+    {
+      title: 'Address',
+      dataIndex: 'address',
+      key: 'address',
+    },
+    {
+      title: 'Tags',
+      key: 'tags',
+      dataIndex: 'tags',
+      filters: [
+        {
+          text: 'Mentee',
+          value: 'mentee'
+        },
+        {
+          text: 'Mentor',
+          value: 'mentor'
+        }
+      ],
+      onFilter: (value, record) => record.tags.includes(value),
+      render: tags => (
+        <>
+          {tags.map(tag => {
+            let color = tag.length > 5 ? 'geekblue' : 'green';
+            if (tag === 'mentee') {
+              color = 'volcano';
+            }
+            return (
+              <Tag color={color} key={tag}>
+                {tag.toUpperCase()}
+              </Tag>
+            );
+          })}
+        </>
+      ),
+    },
+    {
+      title: 'Action',
+      key: 'action',
+      render: (text, record) => (
+        <Space size="middle">
+          <a>Edit</a>
+          <a>Deactivate</a>
+        </Space>
+      ),
+    },
+];
+
+const data = [
+    {
+      key: '1',
+      fname: 'John',
+      lname: 'Brown',
+      age: 32,
+      address: 'New York No. 1 Lake Park',
+      tags: ['mentor'],
```

Comment:
I need to read up more on the tags ant design uses. This is very useful!

Reply:
I agree. They did a great job on this component. There are tons of features.